pom.xml

```diff
@@ -155,7 +155,7 @@
     <openapi.version>2.0</openapi.version>
     <prometheus.version>0.9.0</prometheus.version>
     <protobuf.version>3.17.3</protobuf.version>
-    <quarkus.version>2.1.3.Final</quarkus.version>
+    <quarkus.version>2.2.0.CR1</quarkus.version>
```

Comment:
`2.2.0.Final` should be released today, so we/dependabot can switch to that new version then